jaxrs-clients/src/main/java/com/palantir/remoting/jaxrs/JaxRsClient.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.remoting.jaxrs;
+
+import com.google.common.base.Optional;
+import com.palantir.remoting.ssl.SslConfiguration;
+import java.util.Arrays;
+import java.util.List;
+import javax.net.ssl.SSLSocketFactory;
+
+/**
+ * Static factory methods for producing creating HTTP proxies.
+ * TODO(rfink) Better docs.
+ */
+public final class JaxRsClient {
```

Comment:
Worst case we should only expose our own interceptor object that we can turn into a Feign or Retrofit interceptor...but probably don't even need to expose one right now.